walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -271,4 +491,3 @@ public void notEnoughRoom() {
     Assert.assertEquals(expected, actual);
   }
 }
```

Comment:
I would also add some test cases where the events are overlapping. Some examples:
```
    // Optional:               |------C--------|
    // Events  :       |--A--|     |--B--|
    // Day     : |-----------------------------|
    // Options : |--1--|    
```

```
    // Optional:          |----C------|
    // Events  :       |--A--|     |--B--|
    // Day     : |-----------------------------|
    // Options : |--1--|                 |--2--|
```